Resources/Prototypes/_Goobstation/Catalog/selectable_sets.yml

```
@@ -421,7 +421,7 @@
 - type: selectableSet
   id: ERTBallisticsSet
   name: selectable-set-ert-ballistic-name
-  description: selectable-set-ert-ballistic-description
+  description: tgygbhnjm 
```

```suggestion
  description: selectable-set-ert-ballistic-description
```
Somehow, I don't think you meant to do this.